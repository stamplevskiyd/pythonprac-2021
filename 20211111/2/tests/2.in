print("It is a triangle, don't trust this program")
(True, 1), (0, 0.0), (1.0, False)
(0, 0), (3, 0), (0, 4)
