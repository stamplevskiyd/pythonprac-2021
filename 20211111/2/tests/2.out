It is a triangle, don't trust this program
Invalid input
Not a triangle
6.00
